style.css

```diff
@@ -0,0 +1,651 @@
+}
+
+.nav-link {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+
+  /* identical to box height, or 133% */
+  display: flex;
+  align-items: center;
+  text-align: center;
+  text-decoration: none;
+
+  /* Neutrals/N600 */
+  color: #fff;
+}
+
+.head-page {
+  background-image: url('images/white-painted-wall-texture-background.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.info-page {
+  display: flex;
+  flex-direction: column;
+}
+
+.Greetings {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  color: #ec5242;
+  margin-left: 4%;
+  margin-top: 17%;
+}
+
+.title {
+  font-family: 'cocogoose_letterpressregular', sans-serif;
+  font-weight: 900;
+  color: #ec5242;
+  margin-left: 4%;
+  margin-top: -12px;
+}
+
+.description {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  margin-left: 4%;
+}
+
+.date {
+  color: rgb(85, 83, 83);
+  font-family: 'Lato', sans-serif;
+  font-size: 25px;
+  font-weight: 900;
+  margin-left: 4%;
+}
+
+.tag {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  margin-left: 4%;
+  margin-bottom: 105px;
+}
+
+.main {
+  background-image: url('images/180.jpg');
+  background-position: 100% 100%;
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+}
+
+.recipes {
+  color: rgb(85, 83, 83);
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  margin-top: -10px;
+}
+
+.recipe-block {
+  display: flex;
+  background-color: rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(10px);
+  margin: 10px;
+  padding: 20px;
+}
+
+.category {
+  color: #272a31;
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 15px;
+  margin-left: 9%;
+  margin-top: 3%;
+}
+
+.recipe-text {
+  margin-left: 4%;
+  margin-top: 3%;
+}
+
+button.recipe_btn {
+  padding: 20px 55px;
+  width: fit-content;
+  border: none;
+  background: #ec5242;
+  margin-left: 35%;
+  margin-top: 10px;
+  margin-bottom: 50px;
+}
+
+.recipe_btn {
+  font-size: 17px;
+  font-weight: 500;
+  margin-top: 10px;
+  transition: 200ms all ease-in-out;
+  color: #fff;
+}
+
+.recipe_btn:hover {
+  background: none;
+}
+
+.recipe_btn:active {
+  background: #727ddf;
+  box-shadow: 5px 5px blue;
+}
+
+.img {
+  width: 68px;
+  height: 25px;
+  display: flex;
+  flex-direction: column;
+  margin-top: 3%;
+}
+
+.main-page a {
+  display: none;
+}
+
+.frecipe {
+  color: rgb(85, 83, 83);
+  font-family: 'Lato', sans-serif;
+  font-size: 20px;
+  display: flex;
+  justify-content: center;
+  margin-top: 0;
+}
+
+.featured {
+  background: #fff;
+  display: grid;
+}
+
+.c-card h2 {
+  margin-top: 0;
+  margin-left: 8%;
+  margin-bottom: 20px;
+  font-family: 'Lato', sans-serif;
+  color: #272a31;
+}
+
```

Comment:
Kindly ensure that the speakers' section has the design as specified by the guide.

Your design:
![image](https://user-images.githubusercontent.com/20554583/190858174-42fe9871-9a5a-4218-87a1-98c6a2981bb1.png)


Design guideline:
![image](https://user-images.githubusercontent.com/20554583/190858192-9a9b2ae4-cdf9-4267-a798-a073254513c2.png)